content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,278 @@
+        }
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Non-containerized environment
+
+#### Setting variables
+Depending on how you build and deploy your services' binaries or executables, you may have several options available for setting `DD_ENV`, `DD_SERVICE`, and `DD_VERSION`. Since you may run one or more services per host, we recommend that these environment variables be scoped to a single process.
+
+You may elect export the environment variables in the command for your executable:
+
+`DD_ENV=<env> DD_SERVICE=<service> DD_VERSION=<version> /bin/my-service`
+
+Or you might use [Chef][12], [Ansible][13], or another orchestration tool to populate a service's systemd or initd configuration file with the `DD` environment variables. That way when the service process is started it will have access to those variables.
+
+#### Configuring traces, logs, and StatsD metrics
+
+{{< tabs >}}
+{{% tab "Traces" %}}
+
+As a best practice, it is recommended to configure your [APM tracer][1] with `DD_ENV` as it keeps the definition of `env` closer to the application that is generating the traces. This method allows the `env` tag to be sourced automatically from a tag in the span metadata.
+
+**Note: There can only be one service per span.** Trace metrics generally have a single service as well. However, if you have a different service defined in your hosts' tags, that configured service tag will show up on all trace metrics emitted from that host.
+
+[1]: /tracing/send_traces/
+{{% /tab %}}
+
+{{% tab "Logs" %}}
+
+If you already have [traces enabled for logs injection][1], it will also inject `env`, `service`, and `version` into your logs.
+
+If you already have logs configuration checks defined similarly to the example below, you **do not** need to define `service`:
+
+```yaml
+logs:
+  - type: file
+    path: "/path/to/your/log.log"
+    source: python
+    log_processing_rules:
+      ...
+    tags:
+      ...
+```
+
+This configuration will allow existing configurations, such as log processing rules or other tags, to continue to work exactly the same.
+
+If you do not wish to use automatic log injection from the APM tracer, you also have the option of [manually injecting these fields into logs][2].
+
+[1]: /tracing/connect_logs_and_traces/
+[2]: /tracing/connect_logs_and_traces/java?tab=log4j2#manual-trace-id-injection
+{{% /tab %}}
+
+{{% tab "StatsD" %}}
+
+Tags are added in an append-only fashion for [custom statsd metrics][1]. For example, if you have two different values for env, the metrics will be tagged with both envs. There is no order in which one tag will override another of the same name.
+
+[1]: /developers/metrics/
+{{% /tab %}}
+
+{{< /tabs >}}
+
+#### Configuring integrations
+
+To unify telemetry across your environment based on integration, configure the `service` tag based on your agent version:
+
+{{< tabs >}}
+{{% tab "Agent > v7.18/v6.18" %}}
+
+Define `service` under `init_config` in your `conf.yaml` file:
```

Comment:
This could actually be integrated in the previous section on how to override it for metrics (after explaining how to do it for logs and traces).

However We should strongly push towards the `init_config` section.
Overriding the service in the `instances` is only for the multiple instances case and might be more for an FAQ article which would link to.

Reply:
@olivielpeau Going off of our conversation yesterday, is what Nils is referring to the best way to configure? Instead of by version, it'd be by the case (single vs. multiple instances)?